src/win/core.c

```diff
@@ -77,17 +77,14 @@ static void uv__crt_invalid_parameter_handler(const wchar_t* expression,
 }
 #endif
 
-static uv_loop_t** uv__loops;
+static uv_loop_t** uv__loops = NULL;
 static int uv__loops_size;
 static int uv__loops_capacity;
 #define UV__LOOPS_CHUNK_SIZE 8
 static uv_mutex_t uv__loops_lock;
 
 static void uv__loops_init() {
   uv_mutex_init(&uv__loops_lock);
-  uv__loops = uv__calloc(UV__LOOPS_CHUNK_SIZE, sizeof(uv_loop_t*));
-  if (!uv__loops)
-    uv_fatal_error(ERROR_OUTOFMEMORY, "uv__malloc");
   uv__loops_size = 0;
   uv__loops_capacity = UV__LOOPS_CHUNK_SIZE;
```

Comment:
Doesn't this represent the size that `uv__loops_size` can grow to before a realloc occurs?